spec/fixtures/saved_searches/get.json

```diff
@@ -9,7 +9,8 @@
               "Filter": "City Eq 'Moorhead' And MlsStatus Eq 'Active' And PropertyType Eq 'A'",
               "ContactIds": [
                 "20100815220615294367000000"
-              ]
+              ],
+              "Provided": false
```

Comment:
This will only be populated if the expansion is used. So the `Provided?` method won't work if you are not using the expansion.